File: QuoteFlow/Content/js/app/models/asset_var_value.js
"use strict";

var $ = require('jquery');
var _ = require('underscore');
var Backbone = require('backbone');
Backbone.$ = $;

/**
 * 
 */
var AssetVarValue = Backbone.Model.extend({
    url: function() {
        return QuoteFlow.RootUrl + "api/assetvarvalue";
    },

    defaults: function() {
        return {
            Id: null,
            AssetId: 0,
            VarValue: "",
            AssetVarId: 0,
            OrganizationId: 0
        }
    }
});

module.exports = AssetVarValue;

File: QuoteFlow.Api/Models/AssetVarValue.cs
namespace QuoteFlow.Api.Models
{
    public class AssetVarValue
    {
        public AssetVarValue() { }

        public AssetVarValue(int assetId, int assetVarId, string value, int organizationId)
        {
            AssetId = assetId;
            AssetVarId = assetVarId;
            VarValue = value;
            OrganizationId = organizationId;
        }

        public int Id { get; set; }

        public int AssetId { get; set; }

        public string VarValue { get; set; }

        public int AssetVarId { get; set; }

        public int OrganizationId { get; set; }
    }
}